docker-compose.yml

```diff
@@ -115,7 +115,9 @@ services:
     networks:
       - aiden_network
     environment:
-      - QDRANT_STORAGE_DIR=/data # Adjust path if needed
+      - QDRANT_STORAGE_DIR=/qdrabt/storage # Adjust path if needed
```

Comment:
there's a typo here, also note that I didn't have the time yesterday to properly test qdrant and redis persistence